### PR TITLE
[enterprise-4.7] fix toc issue

### PR DESCRIPTION
**Pull request by pneedle-rh**
This applies to branch/enterprise-4.7 only.

This issue relates to https://github.com/openshift/openshift-docs/pull/30712.

This PR:

* Ensures that the `toc::[]` line in all assemblies is preceded by one blank line
* Removes unnecessary attributes from `modules/common-attributes.adoc`

This PR does not update `contributing_to_docs/doc_guidelines.adoc`, because that document is updated in the master branch only.

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Thanks!

File: installing/installing_openstack/installing-openstack-load-balancing.adoc [hunk 1]
```diff
@@ -5,4 +5,4 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
\ No newline at end of file
+include::modules/installation-osp-balancing-external-loads.adoc[leveloffset=+1]
```
Comment (pneedle-rh) [inline installing/installing_openstack/installing-openstack-load-balancing.adoc:8]: Where you see the last line in a file being removed and added in this PR, this is the TOC blank line correction script adding new lines to the end of a file where one did not previously exist. It was a side effect of the script, but it is useful.

**Comment by bobfuru** [comment]
LGTM!

**Comment by Preeticp** [comment]
Verified for odo, ODC, tkn, build, Pipelines, gitops and helm components. LGTM.

**Pull request merged**